### DISCUSSION TITLE
Process community rewards cycle: 7/30 -> 8/26

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#205] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 7/30 - 8/26
AlgoRhythMatic 0.11 damo
islandboys.algo 0.06 damo
kerrilija 0.12 damo
Maru the Cat 0.09 damo
moloch10 0.03 damo
prince2486 0.03 damo
TEMITOPE 0.03 damo
TOTAL 0.47 damo
```

Please review and comment if there are any questions or concerns. This period uses a 3x coefficient to multiply measurements from the [Rewards Model tab](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=1130085233) from our spreadsheet.

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by ipaleka** [comment]
@AlgoRhythMatic sorry for the late response, I found something:

- - - - -
Maru the Cat | 2022-07-30 | 2022-08-26 | Discord | # feature-request | https://github.com/asastats/channel/issues/201https://discord.com/channels/906917846754418770/910524302531649566/1011469120182702272 | [F] Feature Request | 2 | 100.00% | 0.019396 |   |   |   |   | 
- - - - -

I suppose you missed the following:

https://github.com/asastats/channel/issues/201#issuecomment-1224165894
> This Reddit DM request occurred unnoticed earlier than the original request so I suggest you split rewards 50:50 (I added F2 mark in order to make things easier to do).

So Maru the Cat and u/Aromatic-Ad3922 should split the rewards for this.

**Reply by AlgoRhythMatic** [reply]
Thank you for the clarification!  I saw it in email alert when you added the comment, but did not catch or recall it while closing it.

Here is the updated list:
```
# NOTE ASA Stats Community rewards 7/30 - 8/26
AlgoRhythMatic 0.11 damo
islandboys.algo 0.06 damo
kerrilija 0.12 damo
Maru the Cat 0.06 damo
moloch10 0.03 damo
prince2486 0.03 damo
TEMITOPE 0.03 damo
u/Aromatic-Ad3922 0.03 damo
TOTAL 0.47 damo
```
